includes/RecentChange.php

```diff
@@ -229,24 +229,28 @@ public function save( $noudp = false ) {
 		if( $wgUseEnotif || $wgShowUpdatedMarker ) {
 			// Users
 			if( $this->mAttribs['rc_user'] ) {
-				$editor = ($wgUser->getId() == $this->mAttribs['rc_user']) ?
-					$wgUser : User::newFromID( $this->mAttribs['rc_user'] );
+				if ( $wgUser->getId() !== $this->mAttribs['rc_user'] ) {
+					$editor = User::newFromID( $this->mAttribs['rc_user'] );
+				}
 			// Anons
 			} else {
 				$editor = ($wgUser->getName() == $this->mAttribs['rc_user_text']) ?
 					$wgUser : User::newFromName( $this->mAttribs['rc_user_text'], false );
 			}
```

Comment:
Can this be simplified to the following? It is so difficult to read, and it is not your fault, Matt :)
```php
$editor = null;
if ( isset( $this->mAttribs['rc_user'] ) && $wgUser->getId() !== $this->mAttribs['rc_user'] ) {
	$editor = User::newFromID( $this->mAttribs['rc_user'] );
} else if ( isset( $this->mAttribs['rc_user_text'] ) ) {
	// Same expression as before ...
}

if ( $editor ) {
...
```